proof/proof.go

```diff
@@ -131,7 +131,9 @@ type Watcher interface {
 // Proof encodes all of the data necessary to prove a valid state transition for
 // an asset has occurred within an on-chain transaction.
 type Proof struct {
-	// PrevOut is the previous on-chain outpoint of the asset.
+	// PrevOut is the previous on-chain outpoint of the asset. This outpoint
+	// is that of the first on-chain input. Outpoints which correspond to
+	// the other inputs can be found in AdditionalInputs.
```

Comment:
The existence of `PrevOut` as a field was/is confusing to me. Because in the context of a multi input spend (merge) we seem to make an arbitrary choice of "the asset" that we're spending, is that correct?

> this is the outpoint where the asset currently resides (or previously).

In the context of a multi input spend, what do you mean by "the asset" here?

It's this confusion/uncertainty that motivated me to add more to the comment to make sure I understand what's going on. I think by "the asset" we mean the first input, and that's what I've tried to capture in the comment.